src/scenes/Home/index.js

```diff
@@ -19,16 +19,16 @@ class Home extends Component {
     }
 
     binance.options({
-      'APIKEY':      props.opts.binance.key,
-      'APISECRET':   props.opts.binance.secret,
-      'recvWindow': 60000
+      APIKEY:      props.opts.binance.key,
+      APISECRET:   props.opts.binance.secret,
+      recvWindow:  60000
     })
   }
 
   componentDidMount() {
     var self = this;
-    binance.websockets.candlesticks(['BTCUSDT'], "1m", function(candlesticks) {
-    	let { e:eventType, E:eventTime, s:symbol, k:ticks } = candlesticks;
+    binance.websockets.candlesticks(['BTCUSDT'], "1m", (candlesticks) => {
```

Comment:
Use single quotes

Reply:
Home page will be changed in another branche. This was just a test page.